wpilibc/src/main/native/cpp/PowerDistribution.cpp

```diff
@@ -87,6 +94,16 @@ double PowerDistribution::GetCurrent(int channel) const {
   return current;
 }
 
+std::vector<double> PowerDistribution::GetAllCurrents() const {
+  int32_t status = 0;
+  int32_t size = this->GetNumChannels();
+  std::vector<double> currents(size,0);
```

Comment:
```suggestion
  std::vector<double> currents(size);
```

No need to specify 0, it will default to that.

Reply:
Sorry, I'm not proficient at C++, I do embedded C at work and on a Java team...